utils/chains.ts

```diff
@@ -19,11 +20,31 @@ const customGnosis = {
   iconBackground: 'none',
 };
 
-export const { chains, publicClient, webSocketPublicClient } = configureChains(
-  [mainnet, customGnosis, polygon, arbitrum, optimism, goerli, sepolia],
-  [
-    infuraProvider({ apiKey: process.env.NEXT_PUBLIC_RPC_KEY || '' }),
-    alchemyProvider({ apiKey: process.env.NEXT_PUBLIC_ALCHEMY_KEY || '' }),
-    publicProvider(),
+export default getDefaultConfig({
+  appName: 'Wrapeth',
+  projectId: process.env.NEXT_PUBLIC_PROJECT_ID || '',
+  chains: [
+    mainnet,
+    customGnosis,
+    polygon,
+    arbitrum,
+    optimism,
+    // goerli,
+    sepolia,
+    base,
+    zora,
+    blast,
   ],
-);
+  transports: {
+    [mainnet.id]: http(),
```

Comment:
need to specify a more specific provider than the public endpoint. there should be a way to provide a fallback like we were doing previously

Reply:
Fixed it. They use [fallback](https://wagmi.sh/core/api/transports/fallback) now